dev/archery/archery/integration/datagen.py

```diff
@@ -743,6 +763,82 @@ class LargeStringColumn(_BaseStringColumn, _LargeOffsetsMixin):
     pass
 
 
+class BinaryViewColumn(PrimitiveColumn):
+
+    def _encode_value(self, x):
+        return frombytes(binascii.hexlify(x).upper())
+
+    def _get_buffers(self):
+        char_buffers = []
+        # a small default char buffer size is used so we get multiple
+        # character buffers without massive arrays
+        DEFAULT_BUFFER_SIZE = 32
```

Comment:
Perhaps `MAX_DATA_BUFFER_SIZE`?

Reply:
But it's not the maximum; a string with more than 32 characters would provoke a buffer longer than 32 in order to store its data (alone)

Reply:
Ah, you're right.